### DISCUSSION TITLE
what code implements "combine exclusive subscription types with other subscription types for consumers"??

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#18299] [General]
https://pulsar.apache.org/docs/next/concepts-messaging/#subscriptions
If you want to achieve both effects simultaneously, combine exclusive subscription types with other subscription types for consumers.

i don't know java code  about "combine exclusive subscription types with other subscription types for consumers"，
has demo java code ?
thank you !

**Comment by codelipenghui** [comment]
@startjava You can only have one subscription type for one subscription. If you want to use different subscription types, you need to create multiple subscriptions.

**Comment by startjava** [comment]
@codelipenghui 
thank you !  That's what it means. I know exactly what I'm talking about. Thank you. Thank you.
